index.css

```diff
@@ -0,0 +1,53 @@
+* {
+  color: rgb(25, 25, 77);
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+}
+
+body {
+  margin-left: 20%;
+  margin-right: 20%;
+  background-color: lightgrey;
+}
+
+#main {
+  padding: 15px 30px;
+}
+
+#title {
+  margin-top: 20px;
+  margin-bottom: 50px;
+  font-size: 48px;
+  font-weight: 700;
+  line-height: 40px;
+  text-align: center;
+}
```

Comment:
This's an example of how you should review your RP according to the [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) 
 kindly remove the `#`. when you're selecting the HTML tags in CSS, automatically the css undestad witch title you selected

Reply:
I don't understand your point here.

Reply:
Anyways, I replace `#title` with its respective representative `h1`, assuming I got your point right.

Reply:
I'm not suggesting you make any changes in your code. I'm just showing you how you can review your codes according to the project requirements [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md)

Reply:
- [ ] What the previous reviewer is trying to point out is that you are to play the role of a reviewer in this project. the goal of the project is for you to learn the essential steps of writing a code review by giving a review of a previous project you've worked on.

Kindly give a review of this project and then submit it for review. You can watch this [video with an example](https://www.loom.com/share/e847c46aa4814534b23b3a8e4dd60d6d) of how to give a review, and follow the [How to give a code review?](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) lesson to learn how.